vw/src/main/python/synapse/ml/vw/VowpalWabbitClassificationModel.py

```diff
@@ -4,29 +4,16 @@
 from synapse.ml.vw._VowpalWabbitClassificationModel import (
     _VowpalWabbitClassificationModel,
 )
+from synapse.ml.vw.VowpalWabbitPythonBase import (
+    VowpalWabbitPythonBaseModel,
+)
 from pyspark.ml.common import inherit_doc
 from pyspark import SparkContext, SQLContext
 from pyspark.sql import DataFrame
 
 
 @inherit_doc
-class VowpalWabbitClassificationModel(_VowpalWabbitClassificationModel):
-    def saveNativeModel(self, filename):
-        """
-        Save the native model to a local or WASB remote location.
-        """
-        self._java_obj.saveNativeModel(filename)
-
-    def getNativeModel(self):
-        """
-        Get the binary native VW model.
-        """
-        return self._java_obj.getModel()
-
-    def getReadableModel(self):
-        return self._java_obj.getReadableModel()
-
-    def getPerformanceStatistics(self):
-        ctx = SparkContext._active_spark_context
-        sql_ctx = SQLContext.getOrCreate(ctx)
-        return DataFrame(self._java_obj.getPerformanceStatistics(), sql_ctx)
+class VowpalWabbitClassificationModel(
```

Comment:
Do you still need this wrapper if it doesent override anything?

Reply:
I needs to get the inheritance VowpalWabbitPythonBaseModel